### DISCUSSION TITLE
Missing Detection Rules in ELK

### DISCUSSION THREAD

**Discussion by RomanRetsen** [Security-Onion-Solutions/securityonion#12840] [Unsupported Versions] [answered]
Hello. 
I've started working with SO and notices that in Kibana there is a missing security menu.
So there is no way to use those detection elk rules (KQL, threshhold, EQL, ES|QL) in SO, right?
Or perhaps it's possible to enable it somehow in the settings?
Thank you.

**Comment by cm-ops** [comment, marked as answer]
Are you talking about Features in Kibana? If so, some are disabled by default.  You can enable them by following this https://docs.securityonion.net/en/2.4/kibana.html#features

**Reply by RomanRetsen** [reply]
You've just made my day. Thanks a million.

**Comment by RomanRetsen** [comment]
You've just made my day. Thanks a million.
